teuthology/test/task/test_install.py

```diff
@@ -264,3 +264,36 @@ def only(self, role):
             },
         }
         assert install.upgrade_remote_to_config(ctx, config) == expected_config
+
+
+    @patch("teuthology.task.install.packaging.get_package_version")
+    @patch("teuthology.task.install.set_rh_deb_repo")
+    def test_rh_install_deb_pkgs(self, m_set_rh_deb_repo, m_get_pkg_version):
```

Comment:
I think it would make sense to place each of these tests in the commit that adds or changes the function they're testing